File: DFRobot_RGBLCD/python/SetCursor/SetCursor.py
import sys
sys.path.append('../')
import rgb1602
import time

lcd=rgb1602.RGB1602(16,2)                               #create LCD object,specify col and row

numRows = 2
numCols = 16
lcd.setRGB(0,100,0)
while True:
  # loop from ASCII 'a' to ASCII 'z':
  for thisLetter in range(ord('a'),ord('z')):
    #lcd.clear()
    # loop over the columns:
    for thisRow in range(0,numRows):
      # loop over the rows:
      for thisCol in range(0,numCols):
        # set the cursor position:
        lcd.setCursor(thisCol,thisRow)
        # print the letter:
        lcd.write(thisLetter)
        time.sleep(0.2)

File: DFRobot_RGBLCD/cpp/SetCursor/SetCursor.cpp
#include "../DFRobot_RGBLCD.h"
#include <iostream>
#include <string>

using namespace std;

int main(){
  const int numRows = 2;
  const int numCols = 16;

  DFRobot_RGBLCD lcd(16, 2);
  // initialize
  lcd.init();
  
  while(1){
    // loop from ASCII 'a' to ASCII 'z':
    for (int thisLetter = 'a'; thisLetter <= 'z'; thisLetter++){
      // loop over the columns:
      for (int thisCol = 0; thisCol < numRows; thisCol++){
        // loop over the rows:
        for (int thisRow = 0; thisRow < numCols; thisRow++){
          // set the cursor position:
          lcd.setCursor(thisRow,thisCol);
          // print the letter:
          lcd.write(thisLetter);
          delay(200);
        }
      }
    }
  }
  return 0;
}

